dd-trace-api/src/main/java/datadog/trace/api/config/GeneralConfig.java

```diff
@@ -50,5 +50,7 @@
   public static final String AZURE_APP_SERVICES = "azure.app.services";
   public static final String INTERNAL_EXIT_ON_FAILURE = "trace.internal.exit.on.failure";
 
+  public static final String MONGO_REACTIVESTREAMS_PROPAGATE_ALL_SINGLERESULTCALLBACK = "mongo.reactivestreams.propagate.all.singleresultcallback";
```

Comment:
I know we aren't consistent with this but I think it's easier to spot typos if we change the property like this:
```suggestion
  public static final String MONGO_REACTIVESTREAMS_PROPAGATE_ALL_SINGLERESULTCALLBACK = "mongo.reactivestreams.propagate.all.single-result-callback";
```

Reply:
I agree that we should make the settings more readable. Also, the long line is too long...